tracker-agent/src/main/configuration/cumulocity.json

```diff
@@ -26,7 +27,7 @@
       "port": 80
 
     },
-    "initialDelaySeconds": 200,
+    "initialDelaySeconds": 10,
```

Comment:
this is different than the develop PR. Do you need to change initialDelaySeconds for liveness and readiness probes?

Reply:
Doesn't matter because the number is arbitrary and cumulocity.json is not used, but changed back.